batch/kueue-intro/team-a-job.yaml

```diff
@@ -35,7 +35,7 @@ spec:
         args: ["10s"] # Sleep for 10 seconds
         resources:
           requests:
-            cpu: "1"
```

Comment:
I thought about this, but it actually wouldn't change the cost much.

This just means that more jobs would be accepted in the 10 CPUs quota. Still, it might be more interesting to see more jobs running at a time.

Reply:
I changed the cpu request to 50m but the Cluster queue still admits 3 Jobs. Maybe we can trim down the other resources as well.

Reply:
yes, that's probably why

Reply:
yes, 1 is minimum.
You could reduce cost by adding time-sharing, but it might complicate the tutorial https://cloud.google.com/kubernetes-engine/docs/concepts/timesharing-gpus

Reply:
hmm, timesharing does not support autopilot for now it seems.